README.md

```diff
@@ -63,34 +33,18 @@ a valuable ally on your side.
 > **WARNING**: It's true, it is easier to run dangerous commands through the CLI
 > than through a GUI. Commands that list files or create directories are not
 > likely to break anything. Commands that remove files or directories, however,
-> _should_ be used with caution. Just like that kitchen knife, sometimes you
-> need something that cuts through steak like a hot knife through butter. But
-> you must make sure your fingers are well clear of the blade when you cut.
+> _should_ be used with caution.
 
 ## Define "Command Line Interface"
 
-A CLI is a text-based conversation with the computer in which we type the
-commands for
-
-* viewing, handling, and manipulating files on your computer
-* launching software
-* working with devices like printers or networks
-
-The CLI asks (or "prompts") the user for a command, the user types it in, and
-then the computer runs the "sentence" that was typed in. It returns output, too
-(where appropriate).
-
-## Describe the Purpose of CLIs
-
-With a CLI, users have wide control over the file system and operating system,
-and the tasks become simple.
-
-For example, you can ask, through the CLI, which files are located on the
-`Desktop`. You can create a new folder or delete it. But on top of
-file-management kinds of activities, you can also find out how busy your CPU is,
-how full your hard drive is, and whether your computer can find a network path
-to `flatironschool.com`. On top of this, you don't have to click through several
-menus to get there!
+A CLI is simply a text-based conversation with the computer in which we type the
+commands for accomplishing tasks. With a CLI, users have wide control over the
```

Comment:
This sentence reads a bit strangely, maybe this instead?

A CLI is a program that allows us to have a text-based conversation with the computer in which we type the
commands for accomplishing tasks.

Reply:
Done! Thanks Ian!